src/styles/_ProductDetail.scss

```diff
@@ -6,14 +6,17 @@
   font-family: v.$font;
   min-height: 100vh;
   height: 100%;
+  width: 100%;
   overflow: auto;
+  background: v.$primary-background-color;
   .details__header {
     background-image: url('../images/detail-horizontal-bg.jpg');
     background-size: cover;
     height: 15rem;
-    display: flex;
-    align-items: center;
-    padding: 3rem;
+    width: 100%;
+    @include m.horizontally-aligned-flex;
+    justify-content: center;
+    padding: 3rem 0;
```

Comment:
I think we should have some padding on the side too. It looks weird touching the sides.

Reply:
Also respectfully disagree with this. The photo is a banner and I think it looks extremely odd if it is not touching the sides.

Reply:
I am with Rachel on this one, sorry Taranveer

Reply:
To clarify: The banner picture should touch the sides. IMO, The text shouldn't touch the border of the picture. Padding would affect the text's distance from the picture's border. Not the picture's border from the screen width. 

Do we still have opposing views? Usually flexible on styling but text touching the sides seems really icky.

Reply:
I think that padding is a good idea because there is padding everywhere else.

Reply:
![Screenshot 2023-09-03 at 7 38 53 PM](https://github.com/EverUse/EverUse-FE/assets/122247155/971fb8a4-38e1-494b-9bd0-620e9cc60ebb)
I'm talking about this. The text touching the sides of the picture.

Reply:
Per our conversation, added max-width: 85% to banner text - looks good at all screen sizes. Thanks for noticing this!